setup.py

```diff
@@ -1,16 +1,50 @@
 #!/usr/bin/env python
 # -*- coding: utf-8 -*-
-import os
 
 from setuptools import (
     setup,
     find_packages,
 )
 
 
-DIR = os.path.dirname(os.path.abspath(__file__))
+deps = {
+    'keyfile': [
+        "eth-utils>=1.3.0,<2",
+        "eth-keys>=0.2.1,<0.3.0",
+        "cytoolz>=0.9.0,<1",
```

Comment:
It appears that the **only** place this is used is within `tests/conftest.py`.  Can you update that to import from `eth_utils.toolz` and remove this dependency?

Reply:
Will do.